src/sass/base/dsf-variables.scss

```diff
@@ -3,9 +3,45 @@
 // // -----------------------------------------------------------------------------
 
 
-// Include any default variable overrides here
+// base colours
+ 
+
```

Comment:
This is a nice way of encapsulating these variable 🙌

Reply:
Thanks Ben